### PR TITLE
h1 fixes for jan-thank-you pages

### PR DESCRIPTION
**Pull request by beccaklam**
Fixed bug so that h1 displays the same on all three versions of the video landing page

### CODE REVIEW

**Comment by TheoChevalier** [comment]
This does not seem to work, it falls back to Georgia font

**Comment by TheoChevalier** [comment]
Ok, I’ve found why it’s not working. @beccaklam you can do that in your patch https://github.com/mozilla/donate.mozilla.org/commit/9811c57aebddbdb639540a317629517ccbd986dd Kind of ugly, but it works for me locally

**Comment by TheoChevalier** [comment]
Actually, we should find another way, http://localhost:3000/fr/jan-thank-you-superhero will work, this won’t http://localhost:3000/fr/jan-thank-you-superhero/ (trailing slash)

**Comment by beccaklam** [comment]
@TheoChevalier It seems to work on my computer, but that's not a good indicator I'm guessing... I'm not exactly sure what we can do. Scott was the one who helped me write that bit as the JS is beyond what I can usually do...

**Comment by cadecairos** [comment]
> Actually, we should find another way, http://localhost:3000/fr/jan-thank-you-superhero will work, this won’t http://localhost:3000/fr/jan-thank-you-superhero/ (trailing slash)

missed this! nice catch.

File: src/pages/index.js [hunk 1]
```diff
@@ -10,7 +10,12 @@ var Index = React.createClass({
     var metaData = this.props.metaData;
     var robots = 'index, follow';
     var googleFonts = "https://fonts.googleapis.com/css?family=Open+Sans:600,400,300,300italic";
-    if (metaData.current_url.indexOf("jan-thank-you") !== -1) {
+    var tyPages = [
+      '/' + this.props.locale + '/jan-thank-you-cute',
+      '/' + this.props.locale + '/jan-thank-you-sincere',
+      '/' + this.props.locale + '/jan-thank-you-superhero'
+    ];
+    if (tyPages.indexOf(metaData.current_url) >= 0) {
```
Comment (cadecairos) [inline src/pages/index.js]: you could try:
```js
if (metadata.current_url.match(/jan-thank-you-(cute|sincere|superhero)/) { /*...*/ }
```

Comment (cadecairos) [inline src/pages/index.js]: which matches with/without trailing slash:

**Comment by TheoChevalier** [comment]
My bad, this alone seem to work (seems I had cache issues):
`if (metaData.current_url.indexOf("jan-thank-you") !== -1) {`

**Comment by beccaklam** [comment]
@cadecairos @TheoChevalier Ok I reverted the code in src/pages/index.js back to

>     if (metaData.current_url.indexOf("jan-thank-you") !== -1) {
>       googleFonts = "https://fonts.googleapis.com/css?family=Roboto+Slab:600,400,300,200,100";
>     }
> 

This is hopefully it!

**Pull request merged**